doc/README.md

```diff
@@ -0,0 +1,22 @@
+# Building `doc`
+
+## `make`
+
+### If you've edited a file
+
+```
+   cd doc
+   make html
+```
+`make` in turn calls `sphinx-build`.
```

Comment:
I'd also point to where the generated `index.html` file is located so people can easily find it.